wix-data/wix-data/guides/Error Codes.md

```diff
@@ -33,18 +33,28 @@ When you use the Wix Data API, you may encounter errors. This list includes the
 |WDE0044| Invalid {function name} parameter value. {function name} parameter must be a String.| Check that the parameter value indicated in the error message is a string.|
 |WDE0052| The current site is in template mode. Please save it to modify data.| To modify collection data on a site created from a template, make sure to [save the site](https://support.wix.com/en/article/wix-editor-save-preview-and-publish-your-site#saving-your-site) first.
 |WDE0053| Internal wixData error: {error message}| An unexpected error occurred. If the error persists, please [contact support](https://www.wix.com/contact).
+|WDE0054| Internal server error. wix-data fails to parse the error it gets provided. | This error occurs when you make an invalid request. Check that the data requests in your code are valid.
```

Comment:
A general comment for everything added here. The middle column is supposed to contain the actual text of the error code. Just making sure that's the case for all the codes you added.

In terms of this particular request, I think we should provide some context if possible. Which functions or types of requests might cause this?